tools/webpack

```diff
@@ -18,7 +23,9 @@ def build_for_prod_or_puppeteer(quiet: bool) -> NoReturn:
     webpack_args += ["../node_modules/.bin/webpack-cli", "--mode=production"]
     if quiet:
         webpack_args += ["--stats=errors-only"]
-    os.execvp(webpack_args[0], webpack_args)
+    env = os.environ
+    env["ZULIP_VERSION"] = ZULIP_VERSION
+    os.execvpe(webpack_args[0], webpack_args, env)
```

Comment:
Despite appearances, this mutates `os.environ`. (I think you meant `os.environ.copy()` or `os.execvpe(…, {**os.environ, ZULIP_VERSION="ZULIP_VERSION"})`.)

A more idiomatic option would be `f"--env=ZULIP_VERSION={ZULIP_VERSION}"`, which gets passed within the `env` object to `webpack.config.ts` ([documentation](https://webpack.js.org/guides/environment-variables/)).